src/main/java/seedu/algobase/logic/commands/FindPlanCommand.java

```diff
@@ -0,0 +1,43 @@
+package seedu.algobase.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.algobase.commons.core.Messages;
+import seedu.algobase.model.Model;
+import seedu.algobase.model.plan.NameContainsKeywordsPredicate;
+
+
+/**
+ * Finds and lists all plans in algobase whose name contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindPlanCommand extends Command {
```

Comment:
I'll implement `findplan` that's similar to `find` later, which supports searching by more fields.